pkg/common/catalog/fill.go

```diff
@@ -361,11 +365,17 @@ func (pf *pluginFiller) fillStruct(t reflect.Type) (reflect.Value, bool) {
 			continue
 		}
 
-		fieldValue, ok := pf.fillInterface(ft.Type)
+		var fieldValue reflect.Value
```

Comment:
This little change is so that I can have an embedded `plugin.Facade` filled by the catalog. This is temporary and will not exist in the refactored catalog system.